lib/schmooze/processor_generator.rb

```diff
@@ -44,7 +44,12 @@ def generate_method(name, code)
       end
 
       def generate_import(import)
-        package, mid, path = import[:package].partition('.')
+        if import[:package][0] == '.' # if it local script else package
```

Comment:
`import[:package].start_with?('.')`

Reply:
I did it like this the first time, but for some reason that I can't now remeber, I changed it to [0]. Dunno.